cpp/dataio/trainingwrite.cpp

```diff
@@ -1015,6 +1031,7 @@ void TrainingDataWriter::writeGame(const FinishedGameData& data) {
             sp->unreducedNumVisits,
             &(sp->policyTarget),
             NULL,
+            0.0, 0.0, 0.0,
```

Comment:
I guess we should add these values to the side position computation as well, since they do also form part of the training data. Check above to see where we basically do the same search we do on game positions for side positions, it should be straightforward to have sp->policySurprise, etc.